cypress/e2e/connectors-instances.cy.ts

```diff
@@ -179,18 +179,62 @@ describe('Connectors page', () => {
     cy.findAllByText('Something went wrong').should('have.length', 0);
   });
 
-  xit('can search for a connector', () => {
+  it('can search for a connector', () => {
```

Comment:
Took the chance to fix and re-introduce this skipped test.

Reply:
awesome!